app/src/main/res/drawable/edittext_rounded_corners.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+
+        <solid android:color="@color/grey_blank" />
+         <corners android:radius="@dimen/dp_10"/>
+
+    </shape>
```

Comment:
Please click on autoformat to avoid this.

Reply:
i am use combination of keyboard CTRL + ALT + L , but it does not empty line. So i clean by myself